addons/calendar/models/calendar_event.py

```diff
@@ -802,12 +798,26 @@ def action_open_calendar_event(self):
     def action_sendmail(self):
         email = self.env.user.email
         if email:
-            for meeting in self:
-                meeting.attendee_ids._send_mail_to_attendees(
-                    self.env.ref('calendar.calendar_template_meeting_invitation', raise_if_not_found=False)
-                )
+            self._send_template_to_attendees('calendar.calendar_template_meeting_invitation')
         return True
 
+    def _send_template_to_attendees(self, template_name, attendees=None, raise_if_not_found=False):
```

Comment:
Wondering if `attendees` should be optional here, imo it makes sense to have it mandatory. Is there a specific reason to make it optional ? We could then also remove the `attendees=` part above